README.md

```diff
@@ -473,21 +474,28 @@ You can disable any of the balena-ads-b services by creating a *Device Variable*
 
 ## Using different radio device types
 
-With balena-ads-b you are able to use a variety of SDRs (software defined radios) and other devices such as the FPGA based Mode-S Beast. The default operating mode is to use an RTL-SDR over USB and no additional configuration is needed for this setup.
+With balena-ads-b you are able to use a variety of SDRs (software defined radios) and other devices such as the FPGA based Mode-S Beast and Airspy. The default operating mode is to use an RTL-SDR over USB and no additional configuration is needed for this setup.
 
-If you are using a Mode-S Beast, bladeRF, HackRF, LimeSDR or SoapySDR then you will need to configure this for the device to work as intended.
+If you are using a Mode-S Beast, Airspy, bladeRF, HackRF, LimeSDR or SoapySDR then you will need to configure this for the device to work as intended.
 
 In order to configure the particular device type you are using, you need to create a *Device Variable* named `RADIO_DEVICE_TYPE`. The possible values are below:
 
 - rtlsdr (this is the default and you do not need to configure this variable if you are using an RTL-SDR)
 - modesbeast
+- airspy
 - bladerf
 - hackrf
 - limesdr
 - soapysdr
 
 For example if you have a Mode-S Beast, you set the `RADIO_DEVICE_TYPE` variable to `modesbeast`. Remember to save the device variable settings after you have updated them. Your device should restart automatically once you configure this and the radio should now work.
 
+If you use Airspy and need to power active antennas or preamplifiers, you can enable bias tee by setting a Device Variable named `AIRSPY_ADSB_BIASTEE` to `true`.
+
+If you have multiple Airspy sticks connected to the same device, you can differentiate between them using their serial numbers. The serial number appears in the console logs when `dump1090-fa` starts up. Copy the serial number and set it as the value of a *Device Variable* named `AIRSPY_ADSB_SERIAL`.
+
+**Important:** If the serial number is in hexadecimal format, prefix it with `0x`, e.g., `0x00B512CD22524212`.
+
```

Comment:
Should we mention all the other configuration options here? For example gain, stats, baistee - maybe by linking to the airspy_adsb list of setting or something?

Reply:
Added this to readme now so resolving